test/connectivity/scripts/connectivityTest.sh

```diff
@@ -68,6 +68,7 @@ function prepare_test_from_artifacts() {
     sed -i -e "s@<TestResultCoordinator.VerificationDelay>@$VERIFICATION_DELAY@g" "$deployment_working_file"
     sed -i -e "s@<TestResultCoordinator.OptimizeForPerformance>@$optimize_for_performance@g" "$deployment_working_file"
```

Comment:
Please create a bug/pbi to track to remove OptimizeForPerformance flag and we should add it in TRC arm32 docker file; same way as EdgeAgent and EdgeHub.

Reply:
Do we have a PBI that is a pool for connectivity test framework improvement?

Reply:
@philipktlin Shouldn't the optimize for performance flag always be set to false for the TRC? My understanding was that these additional logs are not useful.